### ISSUE TITLE
Analytics

### ISSUE THREAD

**Issue by mdrlzy** [ARK-Builders/ARK-Navigator#290]
We need to add analytics to collect warnings.
Matomo is suitable for this since it is open source.
Also it is necessary to add a toggle switch in the settings.
https://github.com/matomo-org/matomo
https://github.com/matomo-org/matomo-sdk-android

**Comment by kirillt** [comment]
Good idea. One of such warning would be requesting closed mutex in `ResourcesIndex` (which is fine in general, but in our case all coroutines working with single index could be sequenced).
